fido/client.py

```diff
@@ -0,0 +1,21 @@
+def _twisted_web_client():
+    from twisted.web import client
+    return client
+
+
+class HTTP11ClientProtocolOverride(_twisted_web_client().HTTP11ClientProtocol):
```

Comment:
Yes `twisted.web.client` is going to be imported when this file is imported. However, the file is only imported within the `get_agent` function, and not imported at the top of any module. Thus, the pre-fork condition should never be reached. The test https://github.com/Yelp/fido/blob/master/tests/0_import_reactor_test.py#L15 verifies that twisted is not initialized at fido import time.